packages/wrangler/src/__tests__/helpers/msw/handlers/deployments.ts

```diff
@@ -52,25 +49,22 @@ export const mswSuccessDeployments = [
 								},
 							},
 						],
-					} as DeploymentListRes,
-				})
+					} as DeploymentListRes)
```

Comment:
I don't think we actually need a cast here, right?

Reply:
👍🏻